libs/core/src/lib/switch/switch.component.scss

```diff
@@ -7,3 +7,11 @@
 .fd-switch__invisible-text {
     display: none !important;
 }
+
+// to prevent the transition when the switch is first rendered
+.fd-switch-no-animate * {
+    -webkit-transition: none !important;
+    -moz-transition: none !important;
+    -ms-transition: none !important;
+    -o-transition: none !important;
```

Comment:
I think the browsers that we support already can use `transition` property without vendor prefixes

Reply:
I am not sure if there is already 100% support across all the browsers, desktop and mobile

Reply:
https://caniuse.com/?search=transition

Reply:
@fkolar we are supporting evergreen browsers, and they perfectly support transition property without vendor prefixes

Reply:
Cool. so we just vendor prefixes and we should be good to go right ?